automatic/nexus-repository-migrator/nexus-repository-migrator.nuspec

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>nexus-repository-migrator</id>
+    <title>Nexus Repository Migrator</title>
+    <version>3.70.2.1</version>
+    <authors>Sonatype</authors>
+    <owners>chocolatey-community, jpruskin</owners>
+    <summary>Migrator for the Sonatype Nexus Repository database.</summary>
+    <description><![CDATA[# Nexus Repository Migrator
+
+## Features
+
+In release 3.71.0, Sonatype Nexus Repository began using H2 as its default embedded database.
+OrientDB is now in Extended Maintenance as defined in our [Sunsetting documentation](https://help.sonatype.com/en/sonatype-sunsetting-information.html).
+Those wishing to upgrade to version 3.71.0 or above will need to migrate to either an H2 or PostgreSQL database.
+See our detailed [help topic on Upgrading to 3.71.0 and beyond](https://help.sonatype.com/en/upgrading-to-nexus-repository-3-71-0-and-beyond.html).
+
+This package contains the migrator tool to migrate a Sonatype Nexus Repository 3.70.1 database from OrientDb to H2 (available to all) or PostgreSQL (available to Pro license holders).
+
+You can either install the package and run the migrator by following the instructions in Notes, or run the script `ConvertFrom-NexusOrientDb` in the tools directory.
+
+If you have upgraded to the `nexus-repository` package version 3.71.0.6 and your installation is failing to run, you can run the `Repair-Nexus371FailedUpgrade` script in the tools directory.
+
+## Package Parameters
+
+This package supports the following parameters:
+
+* `/AttemptMigration` - Attempts to migrate your existing Sonatype Nexus install from OrientDb to a H2 database.
```

Comment:
> That sounds like 'We-will-give-this-a-try-fingers-crossed'

I mean, it was written by me 😅 That said, it will very much _attempt_ - it'll check a bunch of stuff first, and fail before calling the migrator if (for instance) you have less than the recommended amount of memory or disk space. I'll adjust the wording, in any case, to sound more hopeful.

Reply:
The parameter is now `/Migrate`?

Reply:
It is. The text still says 'attempt':

> `/Migrate` - Attempts to migrate your existing Sonatype Nexus install from OrientDb to a H2 database.